packages/commands/src/components/command-menu.js

```diff
@@ -187,45 +183,34 @@ function CommandInput( { isOpen, search, setSearch } ) {
  * @ignore
  */
 export function CommandMenu() {
-	const { registerShortcut } = useDispatch( keyboardShortcutsStore );
 	const [ search, setSearch ] = useState( '' );
-	const isOpen = useSelect(
-		( select ) => select( commandsStore ).isOpen(),
-		[]
-	);
+	const isOpenRef = useRef( false );
+	const isOpen = useSelect( ( select ) => {
+		return ( isOpenRef.current = select( commandsStore ).isOpen() );
+	}, [] );
 	const { open, close } = useDispatch( commandsStore );
 	const [ loaders, setLoaders ] = useState( {} );
 
 	useEffect( () => {
-		registerShortcut( {
-			name: 'core/commands',
-			category: 'global',
-			description: __( 'Open the command palette.' ),
-			keyCombination: {
-				modifier: 'primary',
-				character: 'k',
-			},
-		} );
-	}, [ registerShortcut ] );
+		const handleUserKeyPress = ( e ) => {
+			if ( e.metaKey && e.key === 'k' ) {
+				// Prevent the Save dialog to open
+				e.preventDefault();
 
-	useShortcut(
```

Comment:
This should still use the useShortcut for several reasons: cross browser, cross platform and the ability to edit the shortcode programmatically.

Reply:
@youknowriad I pulled the latest trunk changes and now the hooks works as expected. So I reverted the file in [0096521](https://github.com/WordPress/gutenberg/pull/54515/commits/00965219c58a130567052d144b9e2d712cd8a398)